mrp_bom_current_stock/reports/__init__.py

```diff
@@ -0,0 +1 @@
+from . import report_mrpcurrentstock_xlsx
```

Comment:
add empty endline

Reply:
WIP: Test are needed to ensure location filtering is working correctly.